Actyx/industrial-ui
# Fix broken references from source maps to original ts files

Fix broken references from source maps to original ts files.
By including original ts and tsx files into the `package` folder created by `npm pack`.
We exclude tests, stories, and others to make the package lightweight.

Fix: https://github.com/Actyx/Cosmos/issues/5448

Test it as:

- cd in this project
- rm -rf node_modules
- npm i
- npm run build:prod
- npm pack
- copy tarball created in root folder for https://github.com/Actyx/DemoMachineKit
- in DemoMachineKit run `npm remove @actyx/industrial-ui`
- now install this libraries package by using `npm i actyx-industrial-ui-1.0.5.tgz`, package.json should now contain `"@actyx/industrial-ui": "file:actyx-industrial-ui-1.0.45.tgz"`
- now do `npm i`
- run the app `npm run ui:dashboard:start`
- parcel warnings like `Could not load source file "../src/index.ts" in source map of "../../node_modules/@actyx/industrial` should be gone

IMPORTANT:

When this PR is merged and published on npm, please update DemoMachineKit by using `npm remove @actyx/industrial-ui` and ` npm i @actyx/industrial-ui@latest`.

Example:

![Screen Shot 2020-10-30 at 10 30 05 AM](https://user-images.githubusercontent.com/17195702/97690062-bfcc7500-1a9c-11eb-965c-69e9af2e5db9.png)
